backend/src/yeomanui.ts

```diff
@@ -189,7 +189,16 @@ export class YeomanUI {
       this.promptCount = 0;
       this.gen = (gen as Generator);
       this.gen.destinationRoot(targetFolder);
-      
+	  
+	  const originalPrototype = Object.getPrototypeOf(gen);
+	  const originalGenWriting = _.get(originalPrototype, "writing");
+	  if (!originalGenWriting) {
+		originalPrototype.writing = () => { }
+	  }
+	  this.gen.on("method:writing", () => {
+		this.rpc.invoke("setInGeneratingStep", []);
+	  });
```

Comment:
Is there a real need for empty func ?

	  if (originalGenWriting) {
     	       this.gen.on("method:writing", () => {
		     this.rpc.invoke("setInGeneratingStep", []);
	       });
	  }

Reply:
Some gens do not have this func, but we need it here to catch it's event.
If a gen has this func we do not add nothing and use the original one.
The important part here is the event that is thrown when this func is called.